readthedocs/proxito/README.rst

```diff
@@ -42,6 +42,80 @@ path prefix:
                                     version     |
                                              filename
 
+URL patterns
+------------
+
+URL patterns define how docs are served, and how the path is parsed by proxito.
+
+For example, the default paths used are:
+
+- Multi version project: ``/<language>/<version>/<filename>``
+- Single version project: ``/<filename>``
+- Subproject: ``/projects/<subproject-alias>/<filename>``
```

Comment:
The default for a subproject would include a language & version, right?

```suggestion
- Subproject: ``/projects/<subproject-alias>//<language>/<version>/<filename>``
```

Reply:
So, here filename will be resolved depending if the subproject is a single version or multi version project, filename is meant to represent any of those patterns.